style_guide.md

```diff
@@ -0,0 +1,118 @@
+# Style Guide
+
+## What is this?
+This is a set of guidelines on how to *style* code you plan on contributing to the prog-o-meter repository. You don't have to follow each, or any of them, but it may take your PR longer to get merged if someone else has to rewrite and document your code.
+
+## Why is this useful? 
+When everybody is on the same page, everyone writes better code. The purpose of the prog-o-meter project is too get newcomers out of their shell and excited to start learning, and feel confident contributing to the open source community. The faster everyone starts getting comfortable the better the project will be.
+
+## 1. Quick Start
+These are some quick guidelines and principles that this project follows. 
+
+In general...
+* **Structure** should follow the [PEP 8](https://www.python.org/dev/peps/pep-0008/#introduction) format.
+* **Docstrings** should follow the [Google Python](http://google.github.io/styleguide/pyguide.html?showone=Comments#Comments) format. (Sec. 3)
+* Code and comments should be kept **beginner friendly**. Purpose and diction should be easy to understand. (Sec. 4)
+* When in doubt look around; you should ***STAY CONSISTENT*** with the rest of the project. 
+
+In order to keep things beginner friendly... 
```

Comment:
I suggest we add:
In order to keep things easy to read and beginner friendly...